packages/braintree-integration/src/braintree-paypal-ach/braintree-paypal-ach-payment-strategy.ts

```diff
@@ -0,0 +1,178 @@
+import {
+    InvalidArgumentError,
+    MissingDataError,
+    MissingDataErrorType,
+    NotInitializedError,
+    NotInitializedErrorType,
+    OrderFinalizationNotRequiredError,
+    OrderRequestBody,
+    PaymentArgumentInvalidError,
+    PaymentInitializeOptions,
+    PaymentIntegrationService,
+    PaymentMethod,
+    PaymentMethodFailedError,
+    PaymentRequestOptions,
+    PaymentStrategy,
+    UsBankAccountInstrument,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import { BraintreeUsBankAccount, UsBankAccountSuccessPayload } from '../braintree';
+import BraintreeIntegrationService from '../braintree-integration-service';
+import isBraintreeError from '../is-braintree-error';
+import isUsBankAccountInstrumentLike from '../is-us-bank-account-instrument-like';
+
+import { WithBraintreePaypalAchInitializeOptions } from './braintree-paypal-ach-initialize-options';
+
+export default class BraintreePaypalAchPaymentStrategy implements PaymentStrategy {
+    private paymentMethod?: PaymentMethod;
+    private usBankAccount?: BraintreeUsBankAccount;
+    private mandateText = '';
+
+    constructor(
+        private paymentIntegrationService: PaymentIntegrationService,
+        private braintreeIntegrationService: BraintreeIntegrationService,
+    ) {}
+
+    async initialize(
+        options: PaymentInitializeOptions & WithBraintreePaypalAchInitializeOptions,
+    ): Promise<void> {
+        const { braintreeach } = options;
+        const { mandateText } = braintreeach || {};
+
+        if (!options.methodId) {
+            throw new InvalidArgumentError(
+                'Unable to initialize payment because "options.methodId" argument is not provided.',
+            );
+        }
+
+        if (!braintreeach) {
+            throw new InvalidArgumentError(
+                `Unable to initialize payment because "options.braintreeach" argument is not provided.`,
+            );
+        }
+
+        if (!mandateText) {
+            throw new InvalidArgumentError(
+                'Unable to initialize payment because "options.braintreeach.mandateText" argument is not provided.',
+            );
+        }
+
+        this.mandateText = mandateText;
+
+        await this.paymentIntegrationService.loadPaymentMethod(options.methodId);
+
+        const state = this.paymentIntegrationService.getState();
+
+        this.paymentMethod = state.getPaymentMethodOrThrow(options.methodId);
```

Comment:
I didn't find any places where do you use this.paymentMethod except initialize method. I think, you can make this variable as a `const` instead of class variable